CMakeLists.txt

```diff
@@ -91,8 +91,10 @@ if(WASMEDGE_BUILD_TOOLS)
     # Static tools will link LLVM statically.
     set(WASMEDGE_LINK_LLVM_STATIC ON)
     # Tools will forcefully turn on the static library building.
-    set(WASMEDGE_LINK_TOOLS_STATIC ON)
```

Comment:
Don't we need to set the tools to static now?

Reply:
This was a mistake typo. The modified one is correct.